lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -186,6 +188,18 @@ module Constants
                  :default => nil,
                  :secret => true
 
+    # Whether to collect metrics about the plugin usage. Use configuration
+    # parameter monitoring_type to select a monitoring system you want to use.
```

Comment:
Thb, I don't understand that comment :)

The second sentence is just referring to the different parameter that changes the behavior of this one. It kinda binds them together. Do you want to remove it?